### ISSUE TITLE
Scrubbed Launches

### ISSUE THREAD

**Issue by esper-creative** [r-spacex/SpaceX-API#510]
Been monitoring what happens when a launch is scrubbed due to it currently being a frequent occurrence. Starlink 12 has just been scrubbed, the time is currently 11:51 UTC which is 30 mins after when the launch was scrubbed. The API is currently returning the old time/date. Also looks like the wiki was updated around the same time. https://en.wikipedia.org/w/index.php?title=Starlink&diff=981958506&oldid=981553279. 

Just wondering if this is a bug or something not quite updating quickly enough. Any ways of speeding this up?

Also was wondering if there would be some way we could actually start showing all the scrubbed events for a launch? Feel like this would a useful feature to have. 

Thanks for all your help

**Comment by esper-creative** [comment]
Any update on this? Also wondering if there would be a way to list the number of times a launch date has changed. Just keeping a counter and having it as "launch_attempts".

**Comment by jakewmeyer** [comment]
The job checks the [wiki](https://old.reddit.com/r/spacex/wiki/launches/manifest) every 10 minutes, and it's usually updated in a timely manner.

Happy to add a scrub counter of some sort, but we need to decide what defines a scrub. Dates can change frequently, and it doesn't always mean the launch was scrubbed. So a counter purely based on date changes wouldn't be a very accurate representation of launch attempts.

**Comment by esper-creative** [comment]
Yeah that's a good point. Perhaps a launch_date_ammends array of all the previous dates. Would it also be possible to look at the change log and use the comments that are used when changing data? Thanks for your help 

"launch_date_ammends": [ 
{
"old_date": "2020-12-05T22:30:00.000Z",
"changed_at": "2020-12-04T12:00:00.000Z",
"reason" "Reason for change..."
}
]
